test/SlowTests/Sharding/Issues/RavenDB_19744.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents.Operations;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Sharding.Issues
+{
+    public class RavenDB_19744 : RavenTestBase
+    {
+        public RavenDB_19744(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [RavenFact(RavenTestCategory.Patching | RavenTestCategory.Sharding)]
+        public async Task PatchByQueryOnShardedDbShouldGenerateIdsInTheCorrectShard()
+        {
+            using (var store = Sharding.GetDocumentStore())
+            {
+                await store.Maintenance.SendAsync(new CreateSampleDataOperation());
```

Comment:
this is too heavy, you can create few docs yourself, instead of creating 1k docs and 10k timeseries

Reply:
done